numpyro/mcmc.py

```diff
@@ -650,6 +650,11 @@ class MCMC(object):
     :param constrain_fn: Callable that converts a collection of unconstrained
         sample values returned from the sampler to constrained values that
         lie within the support of the sample sites.
+    :param str chain_method: One of 'parallel' (default), 'sequential', 'vectorized'. The method
```

Comment:
Could you add a note saying that this is an experimental argument that may be removed in a future version without warning? 

nit: `hence allows us collecting samples` --> `hence allowing us to collect samples`.

Reply:
done